be/src/vec/data_types/serde/data_type_ipv6_serde.h

```diff
@@ -67,6 +67,9 @@ class DataTypeIPv6SerDe : public DataTypeNumberSerDe<IPv6> {
                                const cctz::time_zone& ctz) const override;
```

Comment:
what about ipv4?

Reply:
ipv4 is UInt32 , so just use DataTypeNumberSerDe<UInt32>